README.md

```diff
@@ -47,6 +47,7 @@ To view which version of Nutils is currently installed, run:
 
     python3 -m pip show nutils
 
+Nutils can be installed in a Windows machine using WSL environment. If you want to assemble matrices in parallel using nutils on a Windows machine, then WSL is the way to go. Instructions to setup WSL are available [here](https://ubuntu.com/tutorials/install-ubuntu-on-wsl2-on-windows-10#1-overview). After setting up WSL, nutils can be installed using the above instructions. 
```

Comment:
This document uses numbered references and a maximum line width of 80 characters for running text. Could you please use the same style. I'm happy to do this for you if you [allow maintainers to edit your branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).

Reply:
Ah! I get it. Will follow the same.